### PR TITLE
chore: update run-rosetta.sh

### PR DESCRIPTION
**Pull request by kaizencc**
This point of this PR is to streamline the docs build in `run-rosetta.sh`. We now
utilize the npm package [`cdk-generate-synthetic-examples`](https://github.com/cdklabs/cdk-generate-synthetic-examples) owned by yours truly.
We also run `rosetta extract` twice, to properly translate generated examples. We purposefully ignore infused examples for this step.

In addition, we remove the `generate-examples` project

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: pack.sh [hunk 1]
```diff
@@ -40,9 +40,10 @@ scripts/run-rosetta.sh --infuse --pkgs-from $TMPDIR/jsii.txt
 
 # Jsii packaging (all at once using jsii-pacmak)
 echo "Packaging jsii modules" >&2
+rosetta_cache_file=$HOME/.s3buildcache/rosetta-cache.tabl.json
```
Comment (kaizen3031593) [inline pack.sh:43]: should this be an environment variable instead? currently I just copied and pasted from run-rosetta and it seems fragile.

File: pack.sh [hunk 2]
```diff
@@ -40,9 +40,10 @@ scripts/run-rosetta.sh --infuse --pkgs-from $TMPDIR/jsii.txt
...
 $PACMAK \
   --verbose \
-  --rosetta-tablet samples.tabl.json \
+  --rosetta-tablet ${rosetta_cache_file} \
```
Comment (rix0rrr) [inline pack.sh:46]: This should not be necessary anymore here.

File: package.json [hunk 3]
```diff
@@ -20,14 +20,15 @@
...
+    "jsii-diff": "^1.49.0",
+    "jsii-pacmak": "^1.49.0",
+    "jsii-reflect": "^1.49.0",
+    "jsii-rosetta": "^1.49.0",
     "lerna": "^4.0.0",
     "patch-package": "^6.4.7",
     "standard-version": "^9.3.2",
-    "typescript": "~3.9.10"
+    "typescript": "~3.9.10",
+    "cdk-generate-synthetic-examples": "^0.0.8"
```
Comment (rix0rrr) [inline package.json:31]: Can we at least get a `0.1.0` version on this 🤣

File: scripts/run-rosetta.sh [hunk 4]
```diff
@@ -55,45 +55,37 @@ if [[ "${jsii_pkgs_file}" = "" ]]; then
...
     echo "💎 Generating synthetic examples for the remainder" >&2
-    time $scriptdir/../tools/@aws-cdk/generate-examples/bin/generate-examples \
-        $genexample_cache_opts \
-        --append-to samples.tabl.json \
+    time npx cdk-generate-synthetic-examples \
         $(cat $jsii_pkgs_file)
 
+    time $ROSETTA extract \
+        --compile \
+        --verbose \
```
Comment (rix0rrr) [inline scripts/run-rosetta.sh:83]: Did this not need a `--loose` to prevent failing on invalid synthetic examples?

**Review by rix0rrr — APPROVED** [review]
Provisional approval, upon addressing these issues and the investigation work you're going to be doing

File: scripts/run-rosetta.sh (hunk 4 above)
Comment (kaizen3031593) [reply scripts/run-rosetta.sh:83]: Perhaps that would be helpful but instead I made sure that generate-examples (currently) produces all valid examples.

Comment (kaizen3031593) [reply scripts/run-rosetta.sh:83]: for extract, there is only a flag `--fail` that we do not set, so even with invalid examples we won't fail.

File: pack.sh (hunk 2 above)
Comment (kaizen3031593) [reply pack.sh]: removed

**Comment by kaizen3031593** [comment]
The test pipeline is working for this commit, so I'm going to go ahead and merge.

**Pull request merged**